common/components/Footer/index.scss

```diff
@@ -79,8 +79,26 @@
 
   @media screen and (max-width: $grid-float-breakpoint) {
     .row {
-      margin-left: -.5rem;
-      margin-right: -.5rem;
+      margin-left: -0.5rem;
+      margin-right: -0.5rem;
     }
   }
 }
+
+.Modal {
+  color: #000;
+}
+
+.modal_open_button {
```

Comment:
Class names should follow the naming convention as outlined in the readme: https://github.com/MyEtherWallet/MyEtherWallet#styling

Reply:
I'll change it so it complies to suit css